ee/query-service/app/server.go

```diff
@@ -191,6 +193,7 @@ func NewServer(serverOptions *ServerOptions) (*Server, error) {
 	}
 
 	telemetry.GetInstance().SetReader(reader)
+	telemetry.GetInstance().SetSaasOperator(constants.SaasSegmentKey)
```

Comment:
Did you miss this at `pkg/query-service/app/server.go` ?

Reply:
No, `pkg/query-service/app/server.go` is only used in non-ee builds. We don't want to track non-ee builds so didn't add there.